README.md

```diff
@@ -225,6 +225,22 @@ it is recommended not to preinstall containerd in MachineSet or BYOH Windows ins
 WMCO supports using a [cluster-wide proxy](https://docs.openshift.com/container-platform/latest/networking/enable-cluster-wide-proxy.html)
 to route egress traffic from Windows nodes on OpenShift Container Platform.
 
+### Running in a disconnected/airgapped environment
+WMCO supports running in a disconnected environment.
+Please follow the [disconnected mirroring docs](https://docs.openshift.com/container-platform/latest/installing/disconnected_install/index.html)
+in order to mirror and run WMCO on your cluster.
+
+Nodes can be added through both MachineSets and the windows-instances ConfigMap.
+The image specified in MachineSets has an extra requirement of having an [SSH server installed and set to run on boot.](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse)
```

Comment:
> Does the SSH server require pre-configuration with the corresponding public key?

it does not.

This needs to be done as the user-data installs the SSH capability via the internet.

Reply:
Changed wording to make it clear the specific capability needs to be installed

Reply:
> This needs to be done as the user-data installs the SSH capability via the internet.

I feel you should also mention the above (the why) along the SSH server installed requirement (the what).

Have you considered the case where the CU environment has a local WSUS server [1] where the capabilities can be pulled from? 

[1] https://learn.microsoft.com/en-us/windows-server/administration/windows-server-update-services/get-started/windows-server-update-services-wsus

Reply:
> Have you considered the case where the CU environment has a local WSUS server [1] where the capabilities can be pulled from?
No, how would that look like in practice?

Reply:
In practice, the template used for the machineSet could be configured with a local Update Server so that the machines install updates and capabilities from the local server instead of going to the Internet. In this scenario, the regular user-data install process may work as expected, and there is no extra requirement.

Reply:
> The image specified in MachineSets has an extra requirement of having [the OpenSSH.Server~~~~0.0.1.0 Windows capability installed]

The _image_ needs the OpenSSH? Or does the node?

> This is normally installed by WMCO using the Machine's user-data

It is not installed in a disconnected cluster? Because the lack of internet?